tools/unicode_table_generator.d

```diff
@@ -927,9 +927,9 @@ void writeCaseFolding(File sink)
                     assert(v.n >= 1); // meaning that start of bucket is always single char
                 }
                 if (i % 6 == 0) writeln();
-                writef("0x%014X,", FCE(v.seq, v.n, v.size, v.entry_len).x);
+                writef("%016X", FCE(v.seq, v.n, v.size, v.entry_len).x);
```

Comment:
In another PR you had a space in between values, would be nice to have that here too.